projects/sbb-esta/angular-business/src/lib/contextmenu/contextmenu.module.ts

```diff
@@ -0,0 +1,13 @@
+import { CommonModule } from '@angular/common';
+import { NgModule } from '@angular/core';
+import { ButtonModule, DropdownModule } from '@sbb-esta/angular-public';
```

Comment:
I just realized what the problem is and I missed this in the initial review.
Business components/modules should not import from public.
I believe the encountered error is a circular reference issue or something similar.
To solve this, a business variant of the dropdown should be created.
This is fairly simple to do: Compare with [radio-button](https://github.com/SchweizerischeBundesbahnen/sbb-angular/tree/master/projects/sbb-esta/angular-business/src/lib/radio-button)
Once that is done, import the business variant dropdown.
Also remove the ButtonModule import, since I don't think it is needed here.